next-docs/pages/components/carousel.tsx

```diff
@@ -154,6 +164,13 @@ const PageCarousel = () => {
             description:
               'Interval of auto sliding in milliseconds. No auto sliding if undefined',
           },
+          {
+            name: 'isRtl',
+            type: 'boolean',
+            required: false,
+            default: '-',
+            description: 'Set right to left (rtl) view',
+          },
         ]}
```

Comment:
I noticed the isNotSmooth prop was added. Could we add this to the documentation as well?

Reply:
This is only for inner needs, we don't get this prop outside